### DISCUSSION TITLE
Limit iterations per epoch

### DISCUSSION THREAD

**Discussion by austinmw** [open-mmlab/mmdetection#9093] [Q&A]
Hi, how can I set a maximum number of iterations per epoch, either by raw number or maybe by percentage of batches in the epoch?

I'm debugging and would find it helpful to only run a few batches per epoch

(I'm using MMEngine and MMDet 3.x)

**Comment by WassimBouatay** [comment]
I have a similar problem, did you find a solution to do so ?
